// Saida esperada: Nenhum erro. - exit code: 0

int func(){
  float f;
  int i;
  return i;
}

int main(){
  func();
}